BlockchainSdk/Blockchains/TON/TONTransactionBuilder.swift

```diff
@@ -42,9 +44,15 @@ final class TONTransactionBuilder {
     /// - Parameters:
     ///   - amount: Amount transaction
     ///   - destination: Destination address transaction
+    ///   - walletAddress: Address of jetton wallet, required for jetton transaction
     /// - Returns: TheOpenNetworkSigningInput for sign transaction with external signer
-    public func buildForSign(amount: Amount, destination: String, params: TONTransactionParams? = nil) throws -> TheOpenNetworkSigningInput {
-        return try self.input(amount: amount, destination: destination, params: params)
+    public func buildForSign(
+        amount: Amount,
+        destination: String,
+        walletAddress: String? = nil,
+        params: TONTransactionParams? = nil
+    ) throws -> TheOpenNetworkSigningInput {
```

Comment:
А давай лучше переделаем что бы полную тарнзакцию передавать
```suggestion
    public func buildForSign(transaction: Transaction) throws -> TheOpenNetworkSigningInput {
```
И уже из нее все брать, плюс адрес ты можешь взять из `self.wallet.defaultAdress`
Можешь глянуть как `EthereumTransactionBuilder` или `CardanoTransactionBuilder` сделано

Reply:
тут нужен именно jettonWallet адрес, его надо снаружи пробрасывать, переименую для ясности